### PR TITLE
Design docs

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?

Update the gatsby static site to MDX so it can render components inline. Test out some layout ideas and put in some reasonable content.

I also upped node to `8.10` to prevent a nested `eslint` dependency from failing to install. Everything seems to be okay but let me know if you think it's better to find a workaround for this rather than upping node.

## Why?

Test out how this would look!

## Screenies

Screenshot 2019-04-05 at 10 14 50

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,10 +25,10 @@ dist
 jsconfig.json
 target
 guui.zip
-packages/design/gatsby/public
```
Comment (walaura) [inline .gitignore]: i had to move this to `/design` for yarn's submodule system to not freak out

Comment (SiAdcock) [reply .gitignore]: I prefer it anyway! 👍

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-8.9
+8.10
```
Comment (SiAdcock) [inline .nvmrc]: Gosh, we should really bump to v10 at some point!

File: packages/design/gatsby-config.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    siteMetadata: {
+        title: `Title from siteMetadata`,
```
Comment (gtrufitt) [inline packages/design/gatsby-config.js]: Do we want a real title?

Comment (walaura) [inline packages/design/gatsby-config.js]:
```suggestion
        title: `Paper Design System`,
```

**Pull request merged**